web/src/main/java/com/google/accompanist/web/WebView.kt

```diff
@@ -222,19 +221,11 @@ fun WebView(
                 webChromeClient = chromeClient
                 webViewClient = client
             }.also { state.webView = it }
-
-            // Workaround a crash on certain devices that expect WebView to be
-            // wrapped in a ViewGroup.
-            // b/243567497
-            val parentLayout = FrameLayout(context)
-            parentLayout.layoutParams = layoutParams
-            parentLayout.addView(childView)
-
-            parentLayout
         },
         modifier = modifier,
         onRelease = { parentFrame ->
-            val wv = parentFrame.children.first() as WebView
+            // Since FrameLayout is removed,parentFrame is just the WebView
+            val wv = parentFrame
```

Comment:
So the name `parentFrame` was for when it was wrapped in a `FrameLayout`. You can just remove most of this and just call `onDispose(it)`

Reply:
Thanks for your comment! I have removed the unused code and resubmitted the pr.